theme/css/test.css

```diff
@@ -0,0 +1,2 @@
+
+/*# sourceMappingURL=test.css.map */
```

Comment:
Continuing on the point about not using `build-web`, i'm not sure why these files (which really look like autogenerated build results) are being committed, rather than generated in the build/dev phases.

Reply:
This file is generated by a gulp task. Currently this is empty as our sass file is pretty empty but will be useful in the future